README.md

```diff
@@ -84,12 +84,13 @@ Kubernetes version.
 
 #### Compatibility Matrix
 
-| kops version | k8s 1.5.x | k8s 1.6.x | k8s 1.7.x | k8s 1.8.x | k8s 1.9.x |
-|--------------|-----------|-----------|-----------|-----------|-----------|
-| 1.9.x        | Y         | Y         | Y         | Y         | Y         |
-| 1.8.x        | Y         | Y         | Y         | Y         | N         |
-| 1.7.x        | Y         | Y         | Y         | N         | N         |
-| 1.6.x        | Y         | Y         | N         | N         | N         |
+| kops version | k8s 1.5.x | k8s 1.6.x | k8s 1.7.x | k8s 1.8.x | k8s 1.9.x | k8s 1.10.x |
+|--------------|-----------|-----------|-----------|-----------|-----------|------------|
+| 1.10.x       | Y         | Y         | Y         | Y         | Y         | Y          |
```

Comment:
I know the README says `kops is intended to be backward compatible.` but is there a point where we would no longer support a older version of k8s with the newest version of Kops? The idea behind that being that we would then be able to clean up older code. I've also not see anyone testing the older versions (such as 1.5.x) so new code may not actually support those versions?

Just a passing thought.

Reply:
It's a balance between the cost of carrying the old branches vs breaking people's clusters (even if those clusters are running versions of k8s that are unsupported).

So far the cost hasn't been too high, but if we came to a point where the cost was high I could see that we would start a deprecation procedure or tell people that they had to use an older version of kops and that they wouldn't be getting bugfixes.

As you say though, it's certainly not a good idea to run e.g. k8s 1.5 with kops today, because it doesn't have nearly the same testing